lib/salus/processor.rb

```diff
@@ -148,12 +153,15 @@ def scan_project
             next
           end
 
-          scanner_threads, copied = run_scanner(config, scanner_class, scanner_name)
-
+          scanner_threads, copied, scanner_reporting_threads =
+            run_scanner(config, scanner_class, scanner_name)
           threads.concat(scanner_threads)
+          reporting_threads.concat(scanner_reporting_threads)
           files_copied.concat(copied)
         end
 
+        reporting_threads.each(&:join)
+        Salus::PluginManager.send_event(:reporting_scanners_ran, @scanners_ran, @report)
```

Comment:
We should probably add a spec to test this new behavior

Reply:
Done